analysis/src/macro_expansion.rs

```diff
@@ -23,7 +20,6 @@ pub struct MacroExpander<T> {
     macros: HashMap<String, MacroDefinition<T>>,
     arguments: Vec<Expression<T>>,
     parameter_names: HashMap<String, usize>,
-    shadowing_locals: HashSet<String>,
```

Comment:
This was unused. Removing it did not break any tests, and we are replacing macros by functions anyway soon.